README.md

```diff
@@ -0,0 +1,203 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The project is using following technologies upto now.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+    <li><a href="https://getbootstrap.com/">BootStrap</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **About**
+> Here you can see the way I work in web-development.
+- **HOme**
+> Home page contians the basic landing information.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> click to see live demo.
+
+- [Live Demo Link](https://dilsherb.github.io/OS-conference/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> You can download and try the source code.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+> 1. A Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### to reach your folder
+> cd my-folder
+### to initialize your folder as git folder
+> git init
+### to clone the project
+> git clone https://github.com/DilsherB/OS-conference.git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Dilsher Balouch
+
+👤 **Dilsher Balouch**
+
+- GitHub: [@DilhserB](https://github.com/DilsherB)
+- Twitter: [@_brilliantMindz](https://twitter.com/_brilliantMindz)
+- LinkedIn: [brilliantMindz](https://linkedin.com/in/brilliantMindz)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Responsive Design**
+- [ ] **menu toggle**
+- [ ] **Dynamic Data in Projects**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://dilsherb.github.io/OS-conference/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project please support by visiting and sharing my website <a href="https://dilsherb.github.io/OS-conference/">Visit my Portfolio</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Special thanks to  Cindy Shin the author of the original design. 
```

Comment:
[Optional] It's recommended to add a link to the design page, so the users can find out who is Cindy Shine.